cdap-app-templates/cdap-etl/cdap-etl-realtime/src/test/java/co/cask/cdap/template/etl/realtime/RealtimeElasticsearchSinkTest.java

```diff
@@ -138,7 +138,7 @@ public void testESSink() throws Exception {
       AdapterManager adapterManager = createAdapter(adapterId, adapterConfig);
 
       adapterManager.start();
-      TimeUnit.SECONDS.sleep(5);
+      TimeUnit.SECONDS.sleep(10);
```

Comment:
@sreevatsanraman For metrics in adapter, we currently don't have a way to expose it. After @gsps1 's changes, we have this capability in Flows. But we need to add that for Workers and expose it through AdapterManager. But eventually we will remove AdapterManager. So should we make the changes?